src/otx/core/data/module.py

```diff
@@ -44,11 +45,7 @@ def num_classes(self) -> int:
 class OTXDataModule(LightningDataModule):
     """LightningDataModule extension for OTX pipeline."""
 
-    def __init__(
-        self,
-        task: OTXTaskType,
-        config: DataModuleConfig | InstSegDataModuleConfig,
-    ) -> None:
+    def __init__(self, task: OTXTaskType, config: DataModuleConfig) -> None:
```

Comment:
As mentioned earlier, I tried to avoid touching the model and data core classes as much as possible.
The change is simply that the InstSegDataModuleConfig is not necessarily separate from the DataModuleConfig.